### ISSUE TITLE
PHP 8.x "Using ${var} in strings is deprecated"

### ISSUE THREAD

**Issue by datverse** [elementor/elementor#21087]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

Hi, 
When I use PHP 8.x (8.1, 8.2) I got log alert:

```
PHP: 2023-01-21 22:04:58 [notice X 0][/usr/local/lsws/domain.com/html/wp-content/plugins/elementor-pro/modules/forms/submissions/database/query.php::103] Using ${var} in strings is deprecated, use {$var} instead [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
```

To HOT FIX this I go to: elementor-pro/modules/forms/submissions/database/query.php
Line 103
Change: $q = "SELECT t_submissions.* FROM `{$this->get_table_submissions()}` t_submissions {$where_sql} {$order_sql} LIMIT ${per_page} OFFSET {$offset}";

To: $q = "SELECT t_submissions.* FROM `{$this->get_table_submissions()}` t_submissions {$where_sql} {$order_sql} LIMIT **_{$per_page}_** OFFSET {$offset}";

Please check and fix this.
Thanks

### Steps to reproduce

1) Go to wp-config.php enable debug
define( 'WP_DEBUG', true );
define( 'WP_DEBUG_DISPLAY', true );

2) Try do something with WP-CLI to got issue:
- Example Update Plugin, Core, Theme

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

### System Info

					
== Server Environment ==
	Operating System: Linux
	Software: LiteSpeed
	MySQL version: MariaDB Server v10.5.18
	PHP Version: 8.1.14
	PHP Memory Limit: 128M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 8M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.1.1
	Site URL: https://domain.com
	Home URL: https://domain.com
	WP Multisite: No
	Max Upload Size: 8 MB
	Memory limit: 40M
	Max Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: 0
	Admin Email: admin@domain.com
	Debug Mode: Active

== Theme ==
	Name: Hello Elementor
	Version: 2.6.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:109.0) Gecko/20100101 Firefox/110.0

== Active Plugins ==

	Elementor
		Version: 3.10.1
		Author: Elementor.com

	Elementor Pro
		Version: 3.10.2
		Author: Elementor.com

== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==

== Elementor Experiments ==
	Optimized DOM Output: Active by default
	Improved Asset Loading: Active by default
	Improved CSS Loading: Active by default
	Inline Font Icons: Inactive
	Accessibility Improvements: Active by default
	Additional Custom Breakpoints: Inactive
	Hide native WordPress widgets from search results: Inactive
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Inactive
	Default to New Theme Builder: Inactive
	Hello Theme Header & Footer: Inactive
	Landing Pages: Inactive
	Nested Elements: Inactive by default
	Lazy Load Background Images: Inactive by default
	Save as Default: Inactive
	Page Transitions: Inactive
	Notes: Active
	Loop: Active
	Loop Carousel: Inactive by default
	Form Submissions: Active by default
	Scroll Snap: Active by default

== Log ==

PHP: 2023-01-18 16:38:54 [notice X 1][/usr/local/lsws/domain.com/html/wp-content/plugins/elementor-pro/plugin.php::494] Creation of dynamic property ElementorPro\Plugin::$updater is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible

== Elementor Pro - Compatibility Tag ==

**Comment by rami-elementor** [comment]
@datverse 

Thank you for taking the time to report this issue.

It will be fixed in the next time release, both in the free version and in pro.

If you see any more php 8 compatibility issues, please ping me directly.

**Comment by datverse** [comment]
> @datverse 
> 
> 
> 
> Thank you for taking the time to report this issue.
> 
> 
> 
> It will be fixed in the next time release, both in the free version and in pro.
> 
> 
> 
> If you see any more php 8 compatibility issues, please ping me directly.

Thanks and love ❤️

**Comment by jamieburchell** [comment]
@rami-elementor Please consider running [`phpstan`](https://phpstan.org/) over the codebase - it will highlight all of these things.

**Comment by nicholaszein** [comment]
Hello @datverse!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro **v3.10.3**! 🥳 

✔️ Please feel free to check it out and update your plugin to the new version!

Cheers 🥂
